src/types/mod.rs

```diff
@@ -249,6 +249,18 @@ macro_rules! pyobject_native_type_info(
                 }
             )?
         }
+
+        impl<$($generics,)*> $crate::impl_::pymodule::PyAddToModule for $name {
+            fn add_to_module(
+                module: &$crate::Bound<'_, $crate::types::PyModule>,
+            ) -> $crate::PyResult<()> {
+                use $crate::types::PyModuleMethods;
+                module.add(
+                    <Self as $crate::PyTypeInfo>::NAME,
+                    <Self as $crate::PyTypeInfo>::type_object_bound(module.py()),
+                )
+            }
+        }
```

Comment:
This must be where `PyCode_Type` is being needed. Is this just for custom exceptions? Maybe we can move it to `create_exception!` macro.

Reply:
Yes but it means you can't reexpose a native Python type in your module. But it's probably not a big deal.

Instead, what about disabling `PyCode` on PyPy, this would remove a significant footgun?

Reply:
>Re-exposing a native Python type is an interesting idea. That could always be done manually in the `#[pymodule_init]`. Maybe that's sufficient?

Yes, it's definitely sufficient but maybe surprising for the user to have exceptions working and the other types not imho so if we can keep it, it's probably better. But I am happy with whatever you prefer.

> Yes disabling PyCode on PyPy would be correct I think.

MR: #3934

Reply:
Anyway, I think to be honest this PR isn't the right discussion for my question, this functionality was already merged and is just being moved here in a way which happened to trigger the warning.

So let's proceed to merge once the PyPy issue is unblocked.